easybuild/main.py

```diff
@@ -176,6 +176,24 @@ def run_contrib_style_checks(ecs, check_contrib, check_style):
     return check_contrib or check_style
 
 
+def copy_ecs_to_target(determined_paths, target_path, prefix=False):
```

Comment:
Doesn't this belong in `easybuild.framework.easyconfig.tools`?

Ideally we don't add much stuff to `main.py` (since then it can only be used here, nowhere else)